#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_not_jpl_package.dpatch by Gerardo Curiel <gcuriel@debianvenezuela.org>
##
## DP: Do not build the JPL (Java) bindings.

@DPATCH@

--- swi-prolog-5.6.47.orig/packages/configure.in
+++ swi-prolog-5.6.47/packages/configure.in
@@ -56,4 +56,4 @@

 if [ -z "$PKG" ]; then
-  PKG="clib cpp odbc table xpce sgml sgml/RDF semweb http chr clpqr nlp ssl pldoc plunit jpl zlib"
+  PKG="clib cpp odbc table xpce sgml sgml/RDF semweb http chr clpqr nlp ssl pldoc plunit zlib"
 fi
